TL improvement suggestion (#370)
07th-mod/higurashi-console-arcs: Update/omot_020.txt

//r入江は、…この期に及んで詩音に条件を突きつけた。…詩音は何をもったいぶるのかと目に怒りを灯す。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "入江は、…この期に及んで詩音に条件を突きつけた。…詩音は何をもったいぶるのかと目に怒りを灯す。",

> 			NULL, "At this late in the game... Irie started pushing demands on Shion. She had a look of rage in her eyes to his roundabout speaking.", Line_Normal);

Reviewer: ```suggestion
			NULL, "At this late in the game... Irie started pushing demands on Shion. His roundabout way of speaking had put a look of rage in her eyes.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#c89a80>入江</color>", NULL, "<color=#c89a80>Irie</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 10, "ps3/s14/10/22100189", 256, TRUE);
	OutputLine(NULL, "「…幸いなことに生命に異常はありませんが、…今の悟史くんには、目に入るものの敵味方、真実と虚実の区別がつかないんです」",

> 			NULL, "\"...Fortunately, there are no life-threatening abnormalities, but... at the moment, Satoshi-kun is unable to distinguish friend from foe, and reality from delusion, so he perceives everyone as hostile to him.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"...Fortunately, there are no life-threatening abnormalities, but... at the moment, Satoshi-kun is unable to distinguish friend from foe, or reality from delusion, so he perceives everyone as hostile to him.\"", Line_Normal);
```

			NULL, "\"If the two of you were going to meet, I wanted both of you to be in good health.", Line_WaitForInput);
	ModPlayVoiceLS(4, 10, "ps3/s14/10/22100211", 256, TRUE);
	OutputLine(NULL, "…しかし、悟史くんの意識は一向に戻らず…結局、話す機会を失ってしまった…」",

> 			NULL, " ...But, Satoshi-kun's self-awareness never returned... and in the end, he and you lost a chance to talk together...\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " ...But, Satoshi-kun's self-awareness never returned... and in the end, the two of you lost the chance to talk to one another...\"", Line_Normal);
```

			NULL, " ...Still, it was my own stupidity that brought you where you are today...", Line_WaitForInput);
	ModPlayVoiceLS(4, 10, "ps3/s14/10/22100214", 256, TRUE);
	OutputLine(NULL, "それを思うと、死にたくなるくらい嫌になります…」",

> 			NULL, " And thinking that, I hate myself enough to want me dead...\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " And thinking that, I hate myself enough to want to die...\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#c89a80>入江</color>", NULL, "<color=#c89a80>Irie</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 10, "ps3/s14/10/22100216", 256, TRUE);
	OutputLine(NULL, "「…病原体の治療法を見つけるため、悟史くんに取り付いたウイルスから…。",
			NULL, "\"..In order to identify a treatment for the pathogens...", Line_WaitForInput);
	ModPlayVoiceLS(4, 10, "ps3/s14/10/22100217", 256, TRUE);
	OutputLine(NULL, "試薬を作り出す研究をしています…」",

> 			NULL, " we're conducting research to create chemical reagents from a virus which infected Satoshi-kun...\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " we're conducting research, creating chemical reagents from the virus that infected Satoshi-kun...\"", Line_Normal);
```

			NULL, "\"Satoko doesn't know.", Line_WaitForInput);
	ModPlayVoiceLS(4, 5, "ps3/s14/05/220500170", 256, TRUE);
	OutputLine(NULL, "…もう全て打ち明けますが、沙都子は一時、雛見沢症候群の末期症状に陥りました」",

> 			NULL, " ...I've decided to tell you all about it. At one point Satoko fell ill with terminal symptoms of Hinamizawa Syndrome.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " ...I think it's time to tell you all about it. At one point Satoko fell ill with terminal symptoms of Hinamizawa Syndrome.\"", Line_Normal);
```

			NULL, "\"No matter how much she hated them, no matter how much she might've wanted to kill them, Satoko would never do that!!", Line_WaitForInput);
	ModPlayVoiceLS(4, 6, "ps3/s14/06/220600121", 256, TRUE);
	OutputLine(NULL, "　勝手なことを言って、あの子のことを悪く言ってごまかしたりするなッッ！！」",

> 			NULL, " Don't just go saying whatever you want. I won't let you slander her to evade my questions!!\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " Don't just go saying whatever you want. I won't let you slander her to dodge my questions!!\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#c89a80>入江</color>", NULL, "<color=#c89a80>Irie</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 10, "ps3/s14/10/22100228", 256, TRUE);
	OutputLine(NULL, "「悟史くんは、…もう白状しますが、叔母を殺したことを私に打ち明けてくれました」",

> 			NULL, "\"Satoshi-kun... I've decided to reveal this. he confessed to me that he had killed his aunt.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"Satoshi-kun... I've decided to reveal this. He confessed to me that he had killed his aunt.\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#5ec69a>魅音</color>", NULL, "<color=#5ec69a>Mion</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 3, "ps3/s14/03/220300040", 256, TRUE);
	OutputLine(NULL, "「沙都子、…村の人間があんたに冷たくしても何もできなくて、本当に悪いと思ってる！",

> 			NULL, "\"Satoko... the villagers treated you so coldly, and yet we couldn't do anything for you. I am really sorry for that!", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"Satoko... the villagers treated you so coldly, and yet we couldn't do anything for you. I'm really sorry about that!", Line_WaitForInput);
```

//rやがて、詩音の気持ちが移ったのか沙都子も声を上げて泣きはじめ、二人はしばらくの間、…ただ抱き合って、泣いていた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "やがて、詩音の気持ちが移ったのか沙都子も声を上げて泣きはじめ、二人はしばらくの間、…ただ抱き合って、泣いていた。",

> 			NULL, "Eventually, Satoko sympathized with Shion's feeling, so she joined Shion in crying out loud for a while longer... just holding one another and crying.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Eventually, Satoko seemed to be moved by Shion's feelings, so she joined Shion in crying out loud for a while longer... just holding one another and crying.", Line_Normal);
```
